command/app.go

```diff
@@ -55,6 +59,14 @@ var app = &cli.App{
 			Name:  "install-completion",
 			Usage: "install completion for your shell",
 		},
+		&cli.StringFlag{
```

Comment:
These flags should not be part of global options since it is not used by all subcommands. It is only required by `cp`. Other commands including `ls`, `du`, `rm` will not use these options. 

Reply:
i thought this might be useful for scenarios where we are running in batch mode and we have multiple copy commands, with all but a few having different source and destination buckets. so user will enter app-level flags and will override for specific ones if desired

Reply:
Hmm. It would be useful but doesn't it mean we should add this flag to run command rather than app options?

Reply:
ooh, that's more logical and the way to go. thanks